### PR TITLE
Add helm and Dockerfile

### PR DESCRIPTION
**Pull request by corest**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.5
```
Comment (fgimenez) [inline Dockerfile]: Why not 3.7 (current stable afaik)?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+ADD ./pv-cleaner-operator /pv-cleaner-operator
```
Comment (fgimenez) [inline Dockerfile]: You could add also a `.dockerignore` file to make the build context smaller and increase build performance, see https://github.com/giantswarm/aws-operator/blob/master/.dockerignore

File: helm/pv-cleaner-operator-chart/templates/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  name: pv-cleaner-operator-configmap
+  namespace: giantswarm
+data:
+  config.yml: |
+    server:
+      listen:
+        address: 'http://0.0.0.0:8000'
+    service:
+      kubernetes:
+        address: ''
```
Comment (fgimenez) [inline helm/pv-cleaner-operator-chart/templates/configmap.yaml]: Out of curiosity, do we need to include these empty values if `inCluster` is true?

Comment (corest) [reply helm/pv-cleaner-operator-chart/templates/configmap.yaml]: Didn't check, but that is the same for all configmap operators templates. So, even if it matters - we will need to update everywhere

Comment (fgimenez) [reply helm/pv-cleaner-operator-chart/templates/configmap.yaml]: ack

File: .gitignore [hunk 4]
```diff
@@ -1 +1,2 @@
-pv-cleaner-operator
\ No newline at end of file
+*
```
Comment (fgimenez) [inline .gitignore]: This should be `.dockerignore` instead of `.gitignore` :)

**Pull request merged**